projects/netgrif-components-core/src/lib/data-fields/multichoice-field/multichoice-autocomplete-field/abstract-multichoice-autocomplete-field-component.component.ts

```diff
@@ -24,20 +24,20 @@ export abstract class AbstractMultichoiceAutocompleteFieldComponentComponent imp
     filteredOptions: Observable<Array<MultichoiceFieldValue>>;
 
     ngOnInit() {
-        this.filteredOptions = this.formControlRef.valueChanges.pipe(
-            startWith(''),
-            map(value => this._filter(value).filter((option) => !this.multichoiceField.value.includes(option.key)))
-        );
+        this.formControlRef.valueChanges.subscribe(newVal => {
```

Comment:
Subscription should be unsubscribed in onDestroy

Reply:
Done